deps.sh

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
-# Uncomment the next line to activate the PiGlow module
-# modprobe i2c-dev && echo 'PiGlow: activating i2c-dev'
+# Activate i2c required by the PiGlow module
+modprobe i2c-dev && echo 'PiGlow: activating i2c-dev'
```

Comment:
@imrehg isn't this just done automatically if one sets the i2c in config.txt?

Reply:
@shaunmulligan interesting point, will have to check.
Though IMHO if we can do from software, instead of `config.txt` changes, that is much better.

* explaining config.txt / Device config changes that people have to apply before they can use the project is worse than doing it automatically
* we do set those in the default config.txt I believe, but that doesn't cover cases when devices are moved between applications for example. Again would need to explain that "make sure that this is set in the config" plus an extra reboot.

Thus as a matter of learning I would be curious to know, in practical sense this feels better even if a config change would do the job too.